alerter/src/main/java/com/tapadoo/alerter/Alerter.java

```diff
@@ -177,7 +181,32 @@ public Alerter setBackgroundColor(@ColorRes final int colorResId) {
 
         return this;
     }
+/**
+     * Set the Alert's Background Colour
+     *
+     * @param color Color of background e.x : Color.parse("#ffffff")
+     * @return This Alerter
+     */
+    public Alerter setBackgroundColorCode(@ColorInt final int color) {
+        if (getAlert() != null && getActivityWeakReference() != null) {
+            getAlert().setAlertBackgroundColor(color);
+        }
+
+        return this;
+    }
+    /**
+     * Set the Alert's Background Colour
+     *
+     * @param color Color of background e.x : "#ffffff"
+     * @return This Alerter
+     */
+    public Alerter setBackgroundColorCode( final String color) {
```

Comment:
Whats the benefit to setting the colour from a string rather than its int value? Could introduce future bugs allowing the user to set the colour from any string they want without validating first.